app/src/main/res/values/strings.xml

```diff
@@ -987,4 +987,10 @@ Do you want to switch to this account?</string>
   <string name="choose_your_master_password">Choose your master password</string>
   <string name="choose_a_unique_and_strong_password_to_keep_your_information_safe">Choose a unique and strong password to keep your information safe.</string>
   <string name="minimum_characters">%1$s characters</string>
+    <string name="email_verification">Email Verification</string>
```

Comment:
I wonder if we can just put this in the `strings_non_localized` since this not a production feature?

Reply:
didnt event know that existed, I was going to ask if making a `debug.xml` would make sense, but if there is already a home for them.